### PR TITLE
dasharo_tools_suite.md: add information about method of disabling Secure Boot

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: Przemyslaw Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

File: docs/common-coreboot-docs/dasharo_tools_suite.md [hunk 1]
```diff
@@ -110,6 +110,28 @@ To access Dasharo Tools Suite:
 * in the Network Boot menu, select the `Dasharo Tools Suite` option,
 * the DTS menu will now appear.
 
+## Common requirements - disabling Secure Boot
+
+Any procedure which affects the firmware should be preceded by controlling
+the Secure Boot status and if it is turned on, turning it off. The enabled
+Secure boot will not only prevent you f rom operating on the firmware, but
```
Comment (TomaszAIR) [inline docs/common-coreboot-docs/dasharo_tools_suite.md:117]:
```suggestion
Secure Boot will not only prevent you from operating on the firmware, but
```

File: docs/common-coreboot-docs/dasharo_tools_suite.md [hunk 2]
```diff
@@ -110,6 +110,28 @@ To access Dasharo Tools Suite:
...
+
+1. Turn off the station, on which you want to test the Dasharo firmware.
+1. Turn the station on and go to the next step immediately.
+1. Hold the `BIOS SETUP KEY` to enter the `BIOS MENU`.
+1. Enter the `Administer Secure Boot` menu using the arrow keys and Enter.
+1. Verify that the `Secure Boot Status` field says `Disabled` - if not,
+  deselect the `Enforce Secure Boot` option using the arrow keys and Enter.
+1. Change the setting of Secure Boot to `Disabled` and press Enter.
+1. Press the `F10` key to open the dialog box.
+1. Press `Enter` to confirm changes and exit f rom the menu.
```
Comment (TomaszAIR) [inline docs/common-coreboot-docs/dasharo_tools_suite.md:130]:
```suggestion
1. Press `Enter` to confirm changes and exit from the menu.
```

File: docs/common-coreboot-docs/dasharo_tools_suite.md [hunk 3]
```diff
@@ -110,6 +110,28 @@ To access Dasharo Tools Suite:
...
+the Secure Boot status and if it is turned on, turning it off. The enabled
+Secure boot will not only prevent you f rom operating on the firmware, but
+you will also not be able to launch DTS.
+
+To check the Secure Boot state:
+
+1. Turn off the station, on which you want to test the Dasharo firmware.
+1. Turn the station on and go to the next step immediately.
+1. Hold the `BIOS SETUP KEY` to enter the `BIOS MENU`.
+1. Enter the `Administer Secure Boot` menu using the arrow keys and Enter.
```
Comment (TomaszAIR) [inline docs/common-coreboot-docs/dasharo_tools_suite.md:125]: @sulewskiprzemyslaw it is called this way on NV41xx? What if we use other stock fw on different machine? Maybe lets say here that we need to localize option for Secure Boot.

File: docs/common-coreboot-docs/dasharo_tools_suite.md [hunk 4]
```diff
@@ -110,6 +110,28 @@ To access Dasharo Tools Suite:
 * in the Network Boot menu, select the `Dasharo Tools Suite` option,
 * the DTS menu will now appear.
 
+## Common requirements - disabling Secure Boot
```
Comment (TomaszAIR) [inline docs/common-coreboot-docs/dasharo_tools_suite.md:113]: @sulewskiprzemyslaw on booting requirements we say that disabling Secure Boot is needed (here https://github.com/Dasharo/docs/blob/master/docs/common-coreboot-docs/dasharo_tools_suite.md#requirements and here https://github.com/Dasharo/docs/blob/master/docs/common-coreboot-docs/dasharo_tools_suite.md#requirements-1) Lets add link to this section there.

**Review by TomaszAIR — CHANGES_REQUESTED** [review]

File: docs/common-coreboot-docs/dasharo_tools_suite.md (hunk 1 above)
Comment (sulewskiprzemyslaw) [reply docs/common-coreboot-docs/dasharo_tools_suite.md:117]: improved

File: docs/common-coreboot-docs/dasharo_tools_suite.md (hunk 2 above)
Comment (sulewskiprzemyslaw) [reply docs/common-coreboot-docs/dasharo_tools_suite.md:130]: improved

File: docs/common-coreboot-docs/dasharo_tools_suite.md (hunk 3 above)
Comment (sulewskiprzemyslaw) [reply docs/common-coreboot-docs/dasharo_tools_suite.md:125]: @TomaszAIR improved. I changed this point to: `Localise and enter the Secure Boot menu using the arrow keys and Enter`

File: docs/common-coreboot-docs/dasharo_tools_suite.md (hunk 4 above)
Comment (sulewskiprzemyslaw) [reply docs/common-coreboot-docs/dasharo_tools_suite.md:113]: OK, link has just been added

**Review by TomaszAIR — APPROVED** [review]

**Pull request merged**